topics/clients.md

```diff
@@ -113,6 +113,44 @@ Query buffer hard limit
 
 Every client is also subject to a query buffer limit. This is a non-configurable hard limit that will close the connection when the client query buffer (that is the buffer we use to accumulate commands from the client) reaches 1 GB, and is actually only an extreme limit to avoid a server crash in case of client or server software bugs.
 
+Client Eviction
```

Comment:
lets mention that it exists only starting redis 7.0

Reply:
:+1: